src/haddock/libs/libplots.py

```diff
@@ -8,6 +8,7 @@
 import plotly.express as px
 import plotly.graph_objects as go
 from plotly.io._utils import plotly_cdn_url
+from plotly.offline.offline import get_plotlyjs
```

Comment:
Please double check if this will affect the renderes - it could be that by using this the plots would only be visible in the browser and maybe not in jupyterlab, cocalc, etc.